File: Javascript/Random/500.js
const findWords = (words) => {
    const result = new Array();

    const a = "qwertyuiop",
        b = "asdfghjkl",
    c = "zxcvbnm"; 
  
    for (let i = 0; i < words.length; i++) {
        const element = words[i].toLowerCase();
      
      const row = a.includes(element[0]) ? a : b.includes(element[0]) ? b : c;
      const f = [...element].every((char) => {
        return row.includes(char)
      })

      if (f) {
        result.push(words[i])
      }

        
    }

   return result
};

const words = ["Hello", "Alaska", "Dad", "Peace"];

console.log(findWords(words));

File: Leetcode/500.py
def findWords(words):
  result = []
  a, b, c = ['qwertyuiop','asdfghjkl', 'zxcvbnm']
  print(a in 'q')
  for word in words:
    element = word.lower()
    row = a if element[0] in a else b if element[0] in b else c
    
    if all(char in row for char in element):
      result.append(word)
      
  return result


  
  
  
words = ["Hello", "Alaska", "Dad", "Peace"]
print(findWords(words))


